proto_converter.go

```diff
@@ -115,9 +115,12 @@ func (converter *protoConverter) toField(key string, value interface{}) *cpb.Key
 		field.Value = &cpb.KeyValue_BoolValue{BoolValue: reflectedValue.Bool()}
 	default:
 		var s string
-		if stringer, ok := value.(fmt.Stringer); ok {
-			s = stringer.String()
-		} else {
+		switch value := value.(type) {
+		case fmt.Stringer:
+			s = value.String()
+		case error:
```

Comment:
Do some `error`s implement `Stringer`? I feel like you might want to reverse the order of these two. But also that's assuming that switch executes in order, which I don't know is guaranteed.

Reply:
Thankfully, `switch` always executes in order. When an error also matches `fmt.Stringer`, the intent is usually to allow the error to print the error message correctly when it's handed to `fmt.Print()`. Since `fmt.Stringer` is specifically for printing, it's probably okay as a default. But my guess is the order almost never matters.